### ISSUE TITLE
mintmanga new url

### ISSUE THREAD

**Issue by rabbi7sf1ght** [tachiyomiorg/tachiyomi-extensions#18134]
### Source information

MintManga 1.4.66

### Source language

Russian 

### Source new URL

https://23.mintmanga.live/

### Other details

there are a lot of problems with russian sources so sorry to bring the mintmanga's url up again. it moved to the new mirrored address recently cause of the shut down of main url
other grouple sources are fine as far as i know

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by MikeZeDev** [comment]
https://mintmanga.live/ works for me, no need for the 23?

**Comment by rabbi7sf1ght** [comment]
it doesn't open in russia (with vpn too). creators of the site used the 23 url in their announcement about the issue

**Comment by rabbi7sf1ght** [comment]
they altered it for 24.mintmanga.live

**Comment by helltraitor** [comment]
@rabbi7sf1ght 
> it doesn't open in Russia (with VPN too)

It can be opened with some of them.

<details>
  <summary>This is not promotion, I describe my use case...</summary>

---

I use antizapret free VPN, which is trust by IT community and proxy only websites, that is blocked by government's RosKomNadzor (aka Russian Communication Supervisor).

Antizapret provides vpn via configuration files and [web extension](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch)

The problem is RKN trying to block any probability of accessing the entry point: https://antizapret.prostovpn.org

Note: source code of extension available [here](https://github.com/anticensority/runet-censorship-bypass)
Note: alternatives can be found [here](https://github.com/anticensority/runet-censorship-bypass/wiki/%D0%90%D0%BB%D1%8C%D1%82%D0%B5%D1%80%D0%BD%D0%B0%D1%82%D0%B8%D0%B2%D1%8B-%D0%BD%D0%B0%D1%88%D0%B5%D0%BC%D1%83-%D1%80%D0%B0%D1%81%D1%88%D0%B8%D1%80%D0%B5%D0%BD%D0%B8%D1%8E)

For android is required to import openvpn configuration file (`ovpn`). So that what i'm doing. I'm very tired from orders: what I can and can't access.

Since site with antizapret cannot be accessed from Russia, I put `.ovpn` source here.
[antizapret-tcp.txt](https://github.com/tachiyomiorg/tachiyomi-extensions/files/12734154/antizapret-tcp.txt)

P.S. If you have access (probably yes for any other country), you can check and put more fresh version (updates per one maybe two month)

---

</details>

What about possible extension fixes:
1. Just use `mintmanga.live` since authors decide to resolve issue with blocking
2. Manually update base each time they update their
3. Try to access site with prefix 24, then 25 and so on, keep prefix in extension preference and use it \ update when needed

In my opinion is better to leave as is: `mintmanga.live`. For now VPN required. Those guys don't like to mess up with authors and blocking, so they will fix what they want, and block will be removed. Same for hentaichan, for which one I have my own extension - when site is blocked, it's blocked for long time, imho
